libs/isograph-react/src/core/read.ts

```diff
@@ -371,9 +374,13 @@ function readData<TReadFromStore>(
       }
       case 'Resolver': {
         const usedRefetchQueries = field.usedRefetchQueries;
-        const resolverRefetchQueries = usedRefetchQueries.map(
-          (index) => nestedRefetchQueries[index],
-        );
+        const resolverRefetchQueries = usedRefetchQueries.map((index) => {
+          const resolverRefetchQuery = nestedRefetchQueries[index];
+          if (resolverRefetchQuery == null) {
+            throw new Error('resolverRefetchQuery is null in Resolver');
```

Comment:
Can you please add `this is indicative of a bug in Isograph`?

Reply:
Done!